deployment/terraform/create.go

```diff
@@ -198,6 +259,11 @@ func (t *Terraform) updateConfig(ip string, sshc *ssh.Client, output *terraformO
 		".SqlSettings.DataSourceReplicas":      readerDSN,
 		".TeamSettings.MaxUsersPerTeam":        50000,
 		".TeamSettings.EnableOpenServer":       true,
+		".ClusterSettings.GossipPort":          8074,
+		".ClusterSettings.StreamingPort":       8075,
+		".ClusterSettings.Enable":              true,
+		".ClusterSettings.ClusterName":         "load-test",
```

Comment:
Let's also use the same `ClusterName` as passed from config to make things more consistent.